### PR TITLE
added webpack to project, added header to html

### PR DESCRIPTION
**Pull request by vikulchik**

### CODE REVIEW

File: client/index.bundle.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/*
```
Comment (serhiirubets) [inline client/index.bundle.js]: bundle js should not be here. 
You should keep all build files in build folder and put it to gitignore

File: client/index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <div class="user-section">
+        <a href="#" class="login">Ввойти</a>
+        <div class="user-section__login-true">
+            <a href="#" class="my-account">Мой аккаунт</a>
+            <a href="#" class="log-out">Выйти</a>
+            <a href="#" class="is-admin">Добавить новый фильм</a>
+        </div>
+    </div>
+</header>
+<script src="index.bundle.js"></script>
```
Comment (serhiirubets) [inline client/index.html]: Pls, use HtmlWebpackPlugin instead of 24 line.
[link](https://webpack.js.org/plugins/html-webpack-plugin/)

Comment (vikulchik) [reply client/index.html]: done

File: client/index.bundle.js (hunk 1 above)
Comment (vikulchik) [reply client/index.bundle.js]: done

**Pull request merged**